docroot/themes/custom/twentyseventeen/sass/components/nav/_nav.scss

```diff
@@ -54,6 +54,11 @@
         overflow: visible;
       }
     }
+    
+    &--second-level,
+    &--third-level {
+      z-index: 1;
```

Comment:
Let's bump this to a higher value just to protect ourselves in the future.  Make it say 500.